pallets/asset/src/checkpoint/mod.rs

```diff
@@ -161,9 +161,9 @@ decl_storage! {
         /// Every schedule-originated checkpoint maps its ID to its due time.
         /// Every checkpoint manually created maps its ID to the time of recording.
         ///
-        /// (checkpoint ID) -> checkpoint timestamp
+        /// (ticker) -> (checkpoint ID) -> checkpoint timestamp
         pub Timestamps get(fn timestamps):
-            map hasher(twox_64_concat) CheckpointId => Moment;
+            double_map hasher(twox_64_concat) Ticker, hasher(twox_64_concat) CheckpointId => Moment;
```

Comment:
Ticker is user input and should use stronger hashing.

```suggestion
            double_map hasher(blake2_128_concat) Ticker, hasher(twox_64_concat) CheckpointId => Moment;
```